apsuite/commissioning_scripts/calc_orbcorr_mat.py

```diff
@@ -32,54 +32,56 @@ def _calc_rfline(self):
             self.model, 'pass_method', 'cavity_pass')[0]
         rffreq = self.model[idx].frequency
         if self.dim == '6d':
-            df = 10
-            self.model[idx].frequency = rffreq + df
+            dfreq = 10
```

Comment:
I propose moving this hard-coded constant to a class constant attribute, at the top of the class implementation, where it is much more visible.

```python 3
class OrbRespMat:
    """."""

    _FREQ_RESP = 10.0  # [Hz]
```